server/controllers/recipes.js

```diff
@@ -94,73 +113,85 @@ export default {
       })
       .then(favoriteRecipeList => res.status(200).json({
         favRecipes: favoriteRecipeList.rows
-      }))
-      .catch(() => res.status(422).json({ error: 'Fav Recipe Not Created yet' }));
+      }));
   },
 
-  update(req, res) {
-    const { userDetail } = req.decoded;
+  /**
+   * function updateRecipe
+   * @param {any} req
+   * @param {any} res
+   * @returns {object} updated recipe
+   */
+  updateRecipe(req, res) {
+    const { id } = req.decoded;
+    const {
+      recipeName,
+      description,
+      ingredients,
+      imageUrl,
+      preptime,
+      servings
+    } = req.body;
     return Recipe
       .find({
         where: {
           id: req.params.recipeId,
         },
-      }).then((Recipefound) => {
-        if (Recipefound && Recipefound.userId === userDetail.id) {
-          return Recipefound
+      })
+      .then((recipeFound) => {
+        if (!recipeFound) {
+          res.status(404).send({ error: 'Recipe not found' });
```

Comment:
To be safe, `return` here would be necessary, or an error/exception would occur when testing `recipeFound.userId === id` below.